docs/source/userManual.rst

```diff
@@ -0,0 +1,17 @@
+|RAT banner|
+
+.. _userManual:
+
```

Comment:
Just for consistency use camelCase for image folder names: UserManual, Chapter1 and Chapter2

Reply:
https://github.com/desktop/desktop/issues/11087

I changed these on my end but they didn't get pushed to GitHub. I fixed it for individual images to make sure all were in camelCase. But, I forgot to check it for the path names. I will fix it now.